requirements-dev.txt

```diff
@@ -7,7 +7,7 @@ mypy>=0.990,<1.1.0; platform_machine != "aarch64" and python_version >= "3.7"
 black==22.3.0
 types-requests==2.28.11.16
 # Prompting libraries needed for testing
-langchain>=0.0.180,<0.1.0; python_version>="3.9"
+langchain>=0.0.180,<0.0.185; python_version>="3.9"
```

Comment:
Where is this needed? (It's not in `setup.cfg`.)

Reply:
What exactly?